### PR TITLE
modules: hal_nordic: Update nrfx to version 2.6.0

### PR DESCRIPTION
**Pull request by nordic-krch**
Update the hal_nordic module revision, to switch to nrfx v2.6.0.

nrfx 2.6.0 includes extended API in nrfx_gpiote which can be
used by the zephyr shim.

Signed-off-by: Krzysztof Chruscinski <krzysztof.chruscinski@nordicsemi.no>

### CODE REVIEW

File: modules/hal_nordic/nrfx/CMakeLists.txt [hunk 1]
```diff
@@ -58,6 +58,7 @@ zephyr_library_sources_ifdef(CONFIG_SOC_NRF5340_CPUNET ${MDK_DIR}/system_nrf5340
 zephyr_library_sources_ifdef(CONFIG_SOC_NRF9160        ${MDK_DIR}/system_nrf9160.c)
 
 zephyr_library_sources(nrfx_glue.c)
+zephyr_library_sources(${NRFX_DIR}/helpers/nrfx_flag32_allocator.c)
```
Comment (nika-nordic) [inline modules/hal_nordic/nrfx/CMakeLists.txt:61]: What about introducing `set(HELPERS_DIR ${NRFX_DIR}/helpers)` and using it here? There might be more helpers with .c files in the future

**Review by nika-nordic — APPROVED** [review]

File: modules/hal_nordic/nrfx/CMakeLists.txt [hunk 2]
```diff
@@ -58,6 +59,7 @@ zephyr_library_sources_ifdef(CONFIG_SOC_NRF5340_CPUNET ${MDK_DIR}/system_nrf5340
 zephyr_library_sources_ifdef(CONFIG_SOC_NRF9160        ${MDK_DIR}/system_nrf9160.c)
```
Comment (anangl) [inline modules/hal_nordic/nrfx/CMakeLists.txt:60]: I think that for now, until the corresponding memory region is excluded from use in Zephyr and `NRF_FICR_NS` becomes actually used, `NRF_SKIP_FICR_NS_COPY_TO_RAM` should be defined in this file for `CONFIG_SOC_NRF5340_CPUAPP` and `CONFIG_SOC_NRF9160`.

**Review by anangl — COMMENTED** [review]
Why are these two commits:
- drivers: pwm: pwm_nrf5_sw: Use runtime resources allocation
- bluetooth: controller: ll_sw: nordic: Rework hw resources files

included in this PR? Are they required for updating nrfx to 2.6.0?

**Comment by nordic-krch** [comment]
PR is based on #40615.

nrfx2.6.0 changed the way how ppi, gpiote resources masks are used. They are used to initialize static variable. Because of that `extern uint32_t flags` cannot be used because it is not considered constant by the compiler. This triggered rework how fixed channels by other modules are used in `nrfx_glue.x`:
- sw pwm dependency is removed here: #40615
- bluetooth controller: Moved all defines which are configuring ppi/gpiote channels to new headers which doesn't have other dependencies (e.g. use of nrfx hal). There are two files with ppi/dppi resources and one file aggregating both and adding gpiote configuration `radio_nrf5_resources.h`. These files can be included by `nrfx_glue.h` because they have no nrfx dependencies.
-  mpsl/radio driver added includes directly in `nrfx_glue.h`

**Comment by nordic-krch** [comment]
>  Are they required for updating nrfx to 2.6.0?

@anangl see https://github.com/zephyrproject-rtos/zephyr/pull/40556#issuecomment-977728674

File: modules/hal_nordic/nrfx/CMakeLists.txt (hunk 2 above)
Comment (nordic-krch) [reply modules/hal_nordic/nrfx/CMakeLists.txt:60]: is this related to nrfx2.6.0 or can be made in separate PR? I'm not familiar with this topic.

Comment (anangl) [reply modules/hal_nordic/nrfx/CMakeLists.txt:60]: It's a new feature that comes with the MDK included in nrfx 2.6.0 (see [here](https://github.com/zephyrproject-rtos/hal_nordic/pull/106/files#diff-5fec5f44d9e363f8a7befe0695b5a91b1f8c9ee3c2d968df350f14de8feb8931R121-R123)), so I think it should be addressed in this PR.

File: modules/hal_nordic/nrfx/CMakeLists.txt [hunk 3]
```diff
@@ -42,6 +42,11 @@ zephyr_compile_definitions_ifdef(CONFIG_NRF_SECURE_APPROTECT_USER_HANDLING
 zephyr_library_compile_definitions_ifdef(CONFIG_NRF_TRACE_PORT
                                  ENABLE_TRACE)
 
+zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF5340_CPUAPP
```
Comment (tejlmand) [inline modules/hal_nordic/nrfx/CMakeLists.txt:45]: Nothing wrong on this line, but the commit message 95616a5d029f0d3ab284bf41ab5719c82e2dfcbf says:
> memory region is ecluded from use.

I assume you mean: `excluded`

**Review by tejlmand — APPROVED** [review]
code looks good.

Minor spelling mistake in a commit message noticed.

**Comment by nordic-krch** [comment]
#40615 is merged now and  is rebased.

@cvinayak can you take a look at rework in bt_ctlr headers?

File: modules/hal_nordic/nrfx/CMakeLists.txt (hunk 2 above)
Comment (nordic-krch) [reply modules/hal_nordic/nrfx/CMakeLists.txt:65]: done

File: modules/hal_nordic/nrfx/nrfx_glue.h [hunk 4]
```diff
@@ -237,39 +272,59 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
...
 #else
 #define NRFX_PPI_CHANNELS_USED_BY_MPSL   0
 #define NRFX_PPI_GROUPS_USED_BY_MPSL     0
 #endif
 
+#if NRF_802154_VERIFY_PERIPHS_ALLOC_AGAINST_MPSL
+BUILD_ASSERT(
+	(NRFX_PPI_CHANNELS_USED_BY_802154_DRV & NRFX_PPI_CHANNELS_USED_BY_MPSL) == 0,
+	"PPI channels used by the Bluetooth controller overlap with those "
+	"assigned to the pwm_nrf5_sw driver.");
```
Comment (anangl) [inline modules/hal_nordic/nrfx/nrfx_glue.h:320]: This copy-pasted message needs to be corrected.

File: modules/hal_nordic/nrfx/nrfx_glue.h [hunk 5]
```diff
@@ -237,39 +272,59 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
...
+
+BUILD_ASSERT(
+	(NRFX_PPI_GROUPS_USED_BY_802154_DRV & NRFX_PPI_GROUPS_USED_BY_MPSL) == 0,
+	"PPI channels used by the Bluetooth controller overlap with those "
+	"assigned to the pwm_nrf5_sw driver.");
```
Comment (anangl) [inline modules/hal_nordic/nrfx/nrfx_glue.h:325]: Same here.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h [hunk 6]
```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef RADIO_NRF5_RESOURCES_H__
```
Comment (anangl) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h:6]:
```suggestion
#ifndef RADIO_NRF5_PPI_RESOURCES_H__
```

File: modules/hal_nordic/nrfx/nrfx_glue.h [hunk 7]
```diff
@@ -237,39 +272,59 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
...
+#include <../src/nrf_802154_peripherals_nrf53.h>
+#define NRFX_PPI_CHANNELS_USED_BY_802154_DRV   NRF_802154_DPPI_CHANNELS_USED_MASK
+#define NRFX_PPI_GROUPS_USED_BY_802154_DRV     NRF_802154_DPPI_GROUPS_USED_MASK
+#else
+#error Unsupported chip family
+#endif
 #else
 #define NRFX_PPI_CHANNELS_USED_BY_802154_DRV   0
 #define NRFX_PPI_GROUPS_USED_BY_802154_DRV     0
 #endif
```
Comment (anangl) [inline modules/hal_nordic/nrfx/nrfx_glue.h:304]:
```suggestion
#endif // defined(CONFIG_NRF_802154_RADIO_DRIVER)
```

**Comment by nordic-krch** [comment]
@anangl comments applied.

**Review by anangl — APPROVED** [review]

**Comment by nordic-krch** [comment]
@kruithofa @cvinayak can you take a look?

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h [hunk 8]
```diff
@@ -4,9 +4,12 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#ifndef NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_DPPI_H
+#define NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_DPPI_H
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h:8]: Do not use guards in the internal controller header files.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h [hunk 9]
```diff
@@ -4,9 +4,12 @@
...
 
 #if defined(CONFIG_SOC_NRF5340_CPUNET) || defined(DPPI_PRESENT)
 
+#include "radio_nrf5_dppi_resources.h"
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h:12]: Move this after the list of standard includes, to after line 20.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h [hunk 10]
```diff
@@ -631,70 +598,4 @@ static inline void hal_radio_group_task_disable_ppi_setup(void)
...
-#endif
-
-#if defined(SW_SWITCH_TIMER_TASK_GROUP_BASE)
-#define HAL_USED_PPI_GROUPS \
-	(BIT(SW_SWITCH_TIMER_TASK_GROUP_BASE) | \
-	 BIT(SW_SWITCH_TIMER_TASK_GROUP_BASE + 1))
-#else
-#define HAL_USED_PPI_GROUPS 0
-#endif
+#endif /* NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_DPPI_H */
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h:601]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h [hunk 11]
```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_DPPI_RESOURCES_H
+#define NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_DPPI_RESOURCES_H
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h:7]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+ *
+ * Note: We already have a PPI where we publish the RATEBOOST event.
+ */
+#define HAL_SW_SWITCH_TIMER_S8_DISABLE_PPI HAL_TRIGGER_RATEOVERRIDE_PPI
+
+#endif /* !CONFIG_BT_CTLR_TIFS_HW */
+
+#endif /* CONFIG_SOC_NRF5340_CPUNET || DPPI_PRESENT */
+
+#endif /* NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_DPPI_RESOURCES_H */
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h:138]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_fem.h [hunk 13]
```diff
@@ -15,6 +15,9 @@
 #include <devicetree.h>
 #include <dt-bindings/gpio/gpio.h>
 
+#ifndef NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_FEM_H__
+#define NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_FEM_H__
+
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_fem.h:20]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_fem.h [hunk 14]
```diff
@@ -58,3 +61,5 @@
 #define HAL_RADIO_GPIO_LNA_POL_INV
 #endif
 #endif	/* HAL_RADIO_GPIO_HAVE_LNA_PIN */
+
+#endif /* NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_FEM_H__ */
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_fem.h:65]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h [hunk 15]
```diff
@@ -4,9 +4,12 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#ifndef RADIO_NRF5_PPI_H__
+#define RADIO_NRF5_PPI_H__
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h:8]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h [hunk 16]
```diff
@@ -4,9 +4,12 @@
...
 
 #if defined(CONFIG_SOC_SERIES_NRF51X) || defined(CONFIG_SOC_COMPATIBLE_NRF52X)
 
+#include "radio_nrf5_ppi_resources.h"
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h:12]: Move this after standard include to line after 14, leave a blank line in between.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h [hunk 17]
```diff
@@ -649,70 +570,4 @@ static inline void hal_radio_sw_switch_ppi_group_setup(void)
...
-#endif
-
-#if defined(SW_SWITCH_TIMER_TASK_GROUP_BASE)
-#define HAL_USED_PPI_GROUPS \
-	(BIT(SW_SWITCH_TIMER_TASK_GROUP_BASE) | \
-	 BIT(SW_SWITCH_TIMER_TASK_GROUP_BASE + 1))
-#else
-#define HAL_USED_PPI_GROUPS 0
-#endif
+#endif /* RADIO_NRF5_PPI_H__ */
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h:573]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h [hunk 18]
```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef RADIO_NRF5_PPI_RESOURCES_H__
+#define RADIO_NRF5_PPI_RESOURCES_H__
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h:7]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h [hunk 19]
```diff
@@ -0,0 +1,169 @@
...
+/* Cancel the SW switch timer running considering S8 timing:
+ * wire the RADIO EVENTS_RATEBOOST event to SW_SWITCH_TIMER TASKS_CAPTURE task.
+ */
+#define HAL_SW_SWITCH_TIMER_S8_DISABLE_PPI 19
+
+#endif /* CONFIG_HAS_HW_NRF_RADIO_BLE_CODED */
+
+#endif /* !CONFIG_BT_CTLR_TIFS_HW */
+
+#endif /* RADIO_NRF5_PPI_RESOURCES_H__ */
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h:169]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_resources.h [hunk 20]
```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_RESOURCES_H
+#define NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_RESOURCES_H
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_resources.h:7]: Remove

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_resources.h [hunk 21]
```diff
@@ -0,0 +1,108 @@
...
+/* Mask with all (D)PPI groups used by the bluetooth controller. */
+#if defined(SW_SWITCH_TIMER_TASK_GROUP_BASE)
+#define BT_CTLR_USED_PPI_GROUPS \
+	(BIT(SW_SWITCH_TIMER_TASK_GROUP_BASE) | \
+	 BIT(SW_SWITCH_TIMER_TASK_GROUP_BASE + 1))
+#else
+#define BT_CTLR_USED_PPI_GROUPS 0
+#endif
+
+#endif /* NORDIC_HAL_NRF5_RADIO_RADIO_NRF5_RESOURCES_H */
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_resources.h:108]: Remove

**Review by cvinayak — CHANGES_REQUESTED** [review]
Just use "Bluetooth: Controller: nordic : Rework hw resource files" as the title for controller changes.

**Comment by nordic-krch** [comment]
@cvinayak thanks for review, comments applied.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h [hunk 22]
```diff
@@ -0,0 +1,164 @@
...
+
+/* PPI channel 27 is pre-programmed with the following fixed settings:
+ *   EEP: RADIO->EVENTS_END
+ *   TEP: TIMER0->TASKS_CAPTURE[2]
+ */
+#define HAL_RADIO_END_TIME_CAPTURE_PPI 27
+
+#else /* defined(CONFIG_BT_CTLR_TIFS_HW) || !defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER) */
+
+#define HAL_RADIO_ENABLE_ON_TICK_PPI 6
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h:42]: Please double verify this change compared to old code

```suggestion
#define HAL_RADIO_ENABLE_ON_TICK_PPI 2
```

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h [hunk 23]
```diff
@@ -0,0 +1,133 @@
...
+ * <index> must be 0 or 1. <offset> must be a valid TIMER CC register offset.
+ */
+#define HAL_SW_SWITCH_GROUP_TASK_DISABLE_PPI_BASE 14
+
+/* Enable the SW Switch PPI Group on RADIO END Event.
+ *
+ * Note: we do not need an additional PPI, since we have already set up
+ * a PPI to publish RADIO END event.
+ */
+#define HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI HAL_SW_SWITCH_TIMER_CLEAR_PPI
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h:106]: Remove this from old code... seems to have been missed?

**Review by cvinayak — CHANGES_REQUESTED** [review]
Sorry to not have done a better review previously. I would like @ppryga  to do another round of review incase I may have missed any more PPI assignments that could be missed/incorrect.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h (hunk 23 above)
Comment (nordic-krch) [reply subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi_resources.h:106]: Removed. Just learned that there is no warning when define is repeated with the same value.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h (hunk 22 above)
Comment (nordic-krch) [reply subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h]: my bad. Checked other defines but did not find other error. Fixed.

**Review by cvinayak — COMMENTED** [review]
LGTM, will wait for @ppryga to approve, I believe he may be having some conflicting changes to his work in progress and would like his thoughts on the changes/review here.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h [hunk 24]
```diff
@@ -0,0 +1,164 @@
...
+#endif /* !CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER */
+
+/* Wire a SW SWITCH TIMER EVENTS_COMPARE[<cc_offset>] event
+ * to a PPI GROUP TASK DISABLE task (PPI group with index <index>).
+ * 2 adjacent PPIs (9 & 10) and 2 adjacent PPI groups are used for this wiring;
+ * <index> must be 0 or 1. <offset> must be a valid TIMER CC register offset.
+ */
+#if defined(CONFIG_SOC_NRF52805)
+#define HAL_SW_SWITCH_GROUP_TASK_DISABLE_PPI_BASE 2
+#else /* -> !defined(CONFIG_SOC_NRF52805) */
```
Comment (ppryga) [inline subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h:124]: nit-pick, we usually write #else this way in controller:
```suggestion
#else /* !CONFIG_SOC_NRF52805 */
```

**Review by ppryga-nordic — APPROVED** [review]
Generally looks ok, nit-picking comments. I didn't tests it on hardware.

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h [hunk 25]
```diff
@@ -0,0 +1,164 @@
...
+ */
+#define HAL_RADIO_DISABLE_ON_HCTO_PPI 22
+
+/* PPI channel 27 is pre-programmed with the following fixed settings:
+ *   EEP: RADIO->EVENTS_END
+ *   TEP: TIMER0->TASKS_CAPTURE[2]
+ */
+#define HAL_RADIO_END_TIME_CAPTURE_PPI 27
+
+#else /* defined(CONFIG_BT_CTLR_TIFS_HW) || !defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER) */
```
[the remark this answers is not in the document]
Comment (nordic-krch) [reply subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h]: done

File: subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h (hunk 24 above)
Comment (nordic-krch) [reply subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h]: done

**Comment by nordic-krch** [comment]
@cvinayak, @ppryga has finished his review. Can you finish your review?

**Review by cvinayak — APPROVED** [review]

**Comment by nordic-krch** [comment]
@carlescufi hal_nordic got nrfx2.6 merged. I removed `DNM`.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**